pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -7023,3 +7023,96 @@ func TestImportJobEventLogging(t *testing.T) {
 		string(jobs.StatusRunning)}
 	backupccl.CheckEmittedEvents(t, expectedStatus, beforeSecondImport.UnixNano(), jobID, "import", "IMPORT")
 }
+
+func TestImportIntCols(t *testing.T) {
```

Comment:
can we rename to `TestImportDefautIntSizeSetting`, thanks!

Reply:
Done.